index.js

```diff
@@ -27,7 +27,7 @@ function download (version, callback) {
   var tarball = path.join(tmpdir, `electron-${version}.tgz`)
 
   var extractor = tar.extract(tmpdir, {
-    ignore: (name) => { return !name.match('docs/')} }
+    ignore: (name) => { return !name.match('docs' + (process.platform === 'win32' ? '\\\\' : '/'))} }
```

Comment:
That's what I thought.  Didn't work though, because this string gets cast to a Regexp.  And a single backslash is invalid in a Regexp.  :'(